extensions/wikia/ArticleVideo/scripts/ooyala-player.js

```diff
@@ -186,10 +193,35 @@ define('ooyala-player', [
 
 					IMAAdsManager.addEventListener('loaded', function (eventData) {
 						var player = html5Player.player,
-							adData = eventData.getAdData();
+							adData = eventData.getAdData(),
+							currentAd = IMAAdsManager.getCurrentAd(),
+							wrapperId;
+
+						if (adData) {
+							options.adTrackingParams.lineItemId = adData.adId;
+							options.adTrackingParams.creativeId = adData.creativeId;
+						}
 
-						options.adTrackingParams.lineItemId = adData.adId;
-						options.adTrackingParams.creativeId = adData.creativeId;
+						if (currentAd) {
+							wrapperId = currentAd.getWrapperAdIds();
+							if (wrapperId.length) {
+								options.adTrackingParams.lineItemId = wrapperId[0];
+							}
+
+							wrapperId = currentAd.getWrapperCreativeIds();
```

Comment:
can we use separate variable for creativeId?

Reply:
Done 👍